### DISCUSSION TITLE
Blockchain, smart contract, solidity

### DISCUSSION THREAD

**Discussion by Ugochi-Eze** [smartcontractkit/full-blockchain-solidity-course-py#1513] [General]
I would be grateful to have tutorial videos that teach and explain how to design a property transfer via smart contract  in the real estate industry.
I would be grateful to have tutorial videos that teach and explain how to design/ build a blockchain based platform into the project execution practice, which initiate payments based on digitally approved work, contractual terms and smart contract actions in the construction industry, also on procurement and supply chain management in the construction industry, and combining BIM and smart assest management with blockchain.

**Comment by ShubhamP48** [comment]
any advice on how to make notes???

**Reply by cromewar** [reply]
I strongly recommend to use Notion, is free and really easy to use.

**Comment by cromewar** [comment]
Hello @Ugochi-Eze 
Unfortunately there is not a lot of content regarding Blockchain and real state, at least not tutorials. Could you share with us more specifically what you want to accomplish ?

**Comment by Ugochi-Eze** [comment]
Hi
Notes?? In terms of what...
